app/assets/stylesheets/application.scss

```diff
@@ -42,3 +42,21 @@
 .home-form {
   color: #ffffff;
 }
+
+.entries {
+  list-style: none;
+  border: 1px solid #999999;
```

Comment:
Color literals like `#999999` should only be used in variable declarations; they should be referred to via variable everywhere else.